docs/en-us/2.0.0/user_doc/guide/installation/pseudo-cluster.md

```diff
@@ -149,6 +149,19 @@ dbname="dolphinscheduler"
 # Registration center address, the address of zookeeper service
 registryServers="localhost:2181"
 ```
+* Modify the environment variable in `dolphinscheduler_env.sh` file which on the 'conf/env' directory (take the relevant software installed under '/opt/soft' as an example)
```

Comment:
Change is good, but I think we should not put these contents in this file. In this folder we should only write something user have to change during their installation time.
Maybe we should add other sections to keep all our configurations in the below place or somewhere. WDYT

![image](https://user-images.githubusercontent.com/15820530/146195240-3d694335-b672-4800-806b-9486abc30861.png)